tests/test_extractor_basic.py

```diff
@@ -0,0 +1,34 @@
+import random
+import pytest
+from decepticonlp.extractor import basic
+
+
+@pytest.mark.parametrize(
+    "words, expected_result",
+    [(["Hey", "There", "Somesh"], [1]), (["Hey", "there"], [1])],
+)
+def test_random_extract(words, expected_result):
+    random.seed(0)
+    random_extractor = basic.RandomImportantWordExtractor()
+    assert random_extractor.extract(words) == expected_result
+
+
+@pytest.mark.parametrize(
+    "words, expected_result", [(["Raju", "Is", "NLP", "GAWD"], [3, 1])]
```

Comment:
Replace with standard test cases. See this from a user's perspective please

Reply:
I vote for Raju is NLP GAWD

Reply:
😂😂